### PR TITLE
autodetect and use HTTP proxy in pivy-importer

### PR DESCRIPTION
**Pull request by busche**
Our company uses an HTTP (and HTTPS) proxy to connect our desktop clients to the internet. The current implementation is extended to check the environment whether or not a proxy is defined. If so, it is used to download the artefacty from pypi.

### CODE REVIEW

**Comment by busche** [comment]
hmm - for whatever reason, Github merges my two commits into one PR ...

Commit 410d2e3 fixes a bug in the following line:
final File repoPath = new File(line.getOptionValue("repo"));
which causes a NPE if no repo command line switch is given.

File: examples/iris-classification/activate [hunk 1]
```diff
@@ -0,0 +1 @@
+/tmp/working/pygradle_busche/examples/iris-classification/build/venv/bin/activate
```
Comment (sixninetynine) [inline examples/iris-classification/activate]: you can remove this file :)

File: examples/iris-classification/build.gradle [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+version=1.0
+
+// we need to define an explicit installation sequence for the dependencies,
+// see issue 75 here: https://github.com/linkedin/pygradle/issues/75 
+project.tasks.findByName('installPythonRequirements').sorted = false
+
+dependencies {
+    python 'pypi:numpy:1.11.2'
+    python 'pypi:pandas:0.19.1'
+	python 'pypi:scipy:0.18.1'
```
Comment (sixninetynine) [inline examples/iris-classification/build.gradle]: indentation mismatch

**Comment by sixninetynine** [comment]
really love the example project!! nice work!

I'll defer to @ethankhall or @zvezdan on the other changes.

**Comment by busche** [comment]
arrg ... wait.

accidently, my commits got mixed up. I'll close this and make a clean PR ...

Sorry, guys ...

**Comment by sixninetynine** [comment]
@busche no worries! looking forward to your new PR 😃

File: examples/iris-classification/activate (hunk 1 above)
Comment (ethankhall) [inline examples/iris-classification/activate]: Add it to the gitignore file :-)

**Pull request closed without merging**